packages/@ember/-internals/metal/lib/computed.ts

```diff
@@ -433,7 +431,7 @@ export class ComputedProperty extends ComputedDescriptor {
     }
 
     if (!this._setter) {
-      return this.clobberSet(obj, keyName, value);
+      assert(`Cannot override the computed property \`${keyName}\` on ${toString(obj)}.`);
     }
```

Comment:
```js
assert(`Cannot override the computed property \`${keyName}\` on ${toString(obj)}.`, this._setter !== undefined);
```

Is likely preferred, as it would cleanly strip the `this._setter` access out of production builds.

Reply:
yeah, dunno what I was thinking… 😅